plugin/Dockerfile

```diff
@@ -1,16 +1,36 @@
-FROM node:16 AS build
+FROM  registry.access.redhat.com/ubi8/nodejs-16 AS build
+
+USER root
+RUN command -v yarn || npm i -g yarn
 
 ADD . /usr/src/app
 WORKDIR /usr/src/app
 RUN yarn install && yarn build
 
-FROM nginx:stable
+FROM registry.access.redhat.com/ubi8/nginx-120
+
+# Add application sources to a directory that the assemble script expects them
+# and set permissions so that the container runs without root access
+USER 0
+
+COPY --from=build /usr/src/app/dist /tmp/src/
+RUN chown -R 1001:0 /tmp/src
+USER 1001
 
-RUN chmod g+rwx /var/cache/nginx /var/run /var/log/nginx
-COPY --from=build /usr/src/app/dist /usr/share/nginx/html
 
 # Have the plugin log its version immediately upon startup
 ARG VERSION
 ARG COMMIT_HASH
-RUN echo "echo 'OpenShift Service Mesh Plugin: Version=[${VERSION}], Commit=[${COMMIT_HASH}]'" > /docker-entrypoint.d/01-log-version.sh
-RUN chmod ug+x /docker-entrypoint.d/01-log-version.sh
```

Comment:
This breaks the feature we just added last week: https://github.com/kiali/openshift-servicemesh-plugin/pull/17

Reply:
Yeah i am checking how to do it but there is not dockerentry in the iamge registry.access.redhat.com/ubi8/nginx-120 taht's why I set this like a draft...researching

Reply:
Let me try to have this in log. If not I don't see bad idea to have an endpoint for this